index.html

```diff
@@ -46,92 +46,8 @@ <h1>Abdel-Khafid SALAOU</h1>
-        <div class="project-card-info deskno">
-            <h3>
-                Hydraulic head loss calculator online
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
-    <div class="project-card three">
-        <a href="#"><img class="cards-img" src="./assets/projects/project_3.svg" alt="My third project"></a>
-        <div class="project-card-info deskno">
-            <h3>
-                Online scientific calculator
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
-    <div class="project-card four">
-        <a href="#"><img class="cards-img" src="./assets/projects/project_4.svg" alt="My fourth project"></a>
-        <div class="project-card-info deskno">
-            <h3>
-                Professional Portfolio
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
-    <div class="project-card deskreduce">
-        <a href="#"><img class="cards-img" src="./assets/projects/project_5.svg" alt="My fifth project"></a>
-        <div class="project-card-info">
-            <h3>
-                Currency converter
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
-    <div class="project-card deskno">
-        <a href="#"><img class="cards-img" src="./assets/projects/project_6.svg" alt="My sixth project"></a>
-        <div class="project-card-info">
-            <h3>
-                Visit counter
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
    </section>
+   <section class="pop-window"></section>
```

Comment:
- [ ] Nice work on making the changes requested. However, there are a few adjustments you need to make to align the `buttons` and text accordingly, kindly shift the button and "previous project" to be aligned with the paragraph above just as in the template. This gives a better user experience from  a user point of view, and helps you stick to requirements  

## View:
![Capture09](https://user-images.githubusercontent.com/50990925/224492484-36e109d7-31d9-4c4e-949d-d9f876be7833.PNG)